### PR TITLE
Prac 08 feedback request

### PR DESCRIPTION
**Pull request by KaiMeloni**
@JarrodPW please review.

### CODE REVIEW

**Comment by JarrodPW** [comment]
Well-written commit messages 👍

File: prac_08/silver_service_taxi_test.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    taxi = SilverServiceTaxi("Test Fancy Taxi", 100, 2)
+    taxi.drive(18)
+    print(taxi)
+    print("$", taxi.get_fare())
```
Comment (lindsaymarkward) [inline prac_08/silver_service_taxi_test.py:8]: How do you know if this worked correctly?

File: prac_08/taxi.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    def __str__(self):
+        """Return a string like a Car but with current fare distance."""
+        return "{}, {}km on current fare, ${:.2f}/km".format(super().__str__(),
+                                                             self.current_fare_distance,
+                                                             self.price_per_km)
+
+    def get_fare(self):
+        """Return the price for the taxi trip."""
+        return self.price_per_km * self.current_fare_distance
```
Comment (lindsaymarkward) [inline prac_08/taxi.py:21]: Missed rounding.

File: prac_08/taxi_simulator.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    print("Taxis are now:")
+    display_taxis(taxis)
+
+
+def display_taxis(taxis):
+    for i, taxi in enumerate(taxis):
+        print("{} - {}".format(i, taxi))
+
+
+def run_tests():
```
Comment (lindsaymarkward) [inline prac_08/taxi_simulator.py:53]: Why is this here? Copied from solutions.

File: prac_08/unreliable_car.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+from prac_08.car import Car
+
+class UnrealibleCar(Car):
+
+    def __init__(self, name, fuel, reliability):
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def drive(self, distance):
+        random_num = randint(1, 100)
```
Comment (lindsaymarkward) [inline prac_08/unreliable_car.py:11]: Don't abbreviate names. Why is 3 fewer characters better?
